docker/Makefile

```diff
@@ -0,0 +1,213 @@
+# classifying images. To use a different model, you should be able
+# to override these variables in mkae.loc
+OPENVINO_MODEL_BASE_URL ?= https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2
+OPENVINO_MODEL ?= resnet50-binary-0001
+
+models :
+	mkdir -p $(DOCKER_DIR)/models/1
+	wget -P $(DOCKER_DIR)/models/1 ${OPENVINO_MODEL_BASE_URL}/${OPENVINO_MODEL}/FP32-INT1/${OPENVINO_MODEL}.bin
+	wget -P $(DOCKER_DIR)/models/1 ${OPENVINO_MODEL_BASE_URL}/${OPENVINO_MODEL}/FP32-INT1/${OPENVINO_MODEL}.xml
+
+clean_models :
+	rm -rf $(DOCKER_DIR)/models
+
+OPENVINO_CONTAINER=openvino/model_server:latest
+OPENVINO_RUN_ARGS += -u $(CONTRACTS_USER_ID):$(CONTRACTS_GROUP_ID)
+OPENVINO_RUN_ARGS += -v $(DOCKER_DIR)/models/:/models/
+OPENVINO_RUN_ARGS += -p 127.0.0.1:9000:9000
+OPENVINO_RUN_COMMAND ?= --model_path /models/ --model_name resnet --port 9000 --shape auto
+
+run_openvino : models
+	docker run $(OPENVINO_RUN_ARGS) -d --name openvino_container $(OPENVINO_CONTAINER) \
+		$(OPENVINO_RUN_COMMAND)
+
+stop_openvino :
+	- docker rm -f openvino_container
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+clean_% :
+	@ docker rmi -f pdo_$*
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+DOCKER_RUN_ARGS += -v $(DOCKER_DIR)/xfer/:/project/pdo/xfer
+DOCKER_RUN_ARGS += --network host
+DOCKER_RUN_COMMAND ?= "stty cols $$(tput cols) rows $$(tput lines) && bash"
+
+stop_contracts :
+	- docker rm -f contracts_container
+
+run_contracts : build_contracts stop_contracts
+	docker run $(DOCKER_RUN_ARGS) -it --name contracts_container pdo_contracts:${CONTRACTS_VERSION} \
+		-c $(DOCKER_RUN_COMMAND)
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+JUPYTER_SERVICE_HOST ?= $(PDO_HOSTNAME)
+JUPYTER_LEDGER ?= $(PDO_LEDGER_URL)
+JUPYTER_MODE ?= build
+
+DOCKER_JUPYTER_COMMAND = "/project/pdo/tools/start_jupyter.sh"
+DOCKER_JUPYTER_COMMAND += -l $(JUPYTER_LEDGER)
+DOCKER_JUPYTER_COMMAND += --service-host $(JUPYTER_SERVICE_HOST)
+DOCKER_JUPYTER_COMMAND += -m $(JUPYTER_MODE)
+
+stop_jupyter :
+	- docker rm -f jupyter_container
+
+run_jupyter : build_contracts stop_jupyter
+	docker run $(DOCKER_RUN_ARGS) -it --name jupyter_container pdo_contracts:${CONTRACTS_VERSION} \
+		$(DOCKER_JUPYTER_COMMAND)
+
+# -----------------------------------------------------------------
+# We need a repository with the source for the branch we are going
+# to build. In theory this could just be a copy of the local source
+# tree but it might contain other artifacts. So instead we clone
+# the contracts with just a single branch in it. The storage and
+# performance requirements are relatively low.
+# -----------------------------------------------------------------
+contracts :
```

Comment:
happy to make the change though there are reasons for either approach. specifically... the pdo_client container already has source from pdo inside it. the contracts respository has another copy. also... we may build the docker containers for PDO in this directory in the future. so... yes, repository is consistent. but its not obviously the best choice.

Reply:
but inside the container the names are anyway different? E.g., pdo is also not called repository?  I was only talking about the host directory name (and related targets)

Reply:
yes. i understand. i changed the name because i specifically wanted to differentiate from any future that involves running the pdo docker makefiles in the same directory in order to build the pdo images (which are necessary for automated testing).